README.md

```diff
@@ -147,6 +147,21 @@ Then you need to edit the `HDDFANCONTROL_ARGS` variable in `/etc/conf.d/hddfanco
 
 You can then start the daemon with `sudo systemctl start hddfancontrol`, see its status with `sudo systemctl status hddfancontrol` and enable automatic startup at boot time with `sudo systemctl enable hddfancontrol`.
 
+## OpenRC service
+
+An OpenRC service file is provided to control the daemon easily.
+If you installed hddfancontrol from a distribution package, you likely already have it installed, otherwise you can install it from the sources of this repository with:
```

Comment:
Are there any OpenRC distros packaging hddfancontrol?

Reply:
Unfortunately not. I'm using Alpine Linux and had to write the startup scripts for OpenRC myself. It may help other users.